src/app.scss

```diff
@@ -10,15 +10,20 @@
 
 html {
 	margin: 0;
-	padding: 0;
+	overflow-y: auto;
+	overflow-x: hidden;
+	height: max-content;
 	font-size: 100%;
 	box-sizing: border-box;
 	overflow-y: scroll;
 }
 
 body {
 	margin: 0;
-	padding: 0;
+	min-height: 100dvh;
+	height: 100%;
+	padding-top: 70px;
+	position: relative;
```

Comment:
what's the purpose for this? why is this in body

Reply:
padding is needed for the navbar.
The rest is needed to fix the issue illustrated below:
![image](https://github.com/ReVanced/revanced-website/assets/56329333/367169f3-79fe-474a-b234-bfdcc74e9af3)

Reply:
Hm okay, I'm not too good with changes like that so I'll ask afn to review this part

Reply:
Wouldn't it be possible to be done in the +page.svelte directly? instead of app.scss